app/views/persons/_desktop.html.erb

```diff
@@ -10,7 +10,7 @@
         <% end %>
       </div>
       <div class="col-8">
-        <h1 class="person-name page-title"><%= @person.first_name %> <%= @person.last_name %></h1>
+        <h1 class="person-name page-title"><%= title @person.first_name + " " + @person.last_name %></h1>
```

Comment:
Please use string interpolation instead of concatenation
`<%= title "#{@person.first_name}  #{@person.last_name}" %>`

It's more idiomatic ruby, plus it is safer against unexpected bad data like `@person.first_name` being nil.

Reply:
got it